packages/@coorpacademy-components/src/molecule/banner/index.js

```diff
@@ -1,19 +1,32 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
-import {NovaSolidInterfaceFeedbackInterfaceAlertDiamond as QuestionIcon} from '@coorpacademy/nova-icons';
+import {
+  NovaSolidInterfaceFeedbackInterfaceAlertDiamond as QuestionIcon,
+  NovaCompositionCoorpacademyValidate as ValidateIcon
+} from '@coorpacademy/nova-icons';
 import Link from '../../atom/button-link';
 import style from './style.css';
 
 const Banner = props => {
-  const {type, message, firstCTA, firstCTALabel, secondCTALabel, secondCTA} = props;
+  const {type, message, firstCTA, firstCTALabel, secondCTALabel, secondCTA, temporary} = props;
   return (
     <div
-      className={classnames(style.banner, type === 'error' && style.errorBanner)}
+      className={classnames(
+        style.banner,
+        type === 'error' && style.errorBanner,
+        type === 'success' && temporary && style.successTemporaryBanner,
+        type === 'error' && temporary && style.errorTemporaryBanner
+      )}
       title={message}
     >
       <div data-name={`${type}-banner-message`} className={style.message}>
-        <QuestionIcon className={style.icon} />
+        {type === 'success' ? (
+          <ValidateIcon className={classnames(style.icon, temporary && style.temporaryIcon)} />
+        ) : (
+          <QuestionIcon className={classnames(style.icon, temporary && style.temporaryIcon)} />
+        )}
+
         {message}
```

Comment:
```
const ICONS : {
  success: ValidateIcon,
  error: QuestionIcon,
  information: QuestionIcon
}
```
`const Icon = ICONS[type];`

```suggestion
       <Icon className={classnames(style.icon, temporary && style.temporaryIcon)} />
       {message}
```

Reply:
`type: PropTypes.oneOf(keys(ICON)),`